hilti/include/rt/types/stream.h

```diff
@@ -565,15 +572,15 @@ class View {
     bool isOpenEnded() const { return ! _end.has_value(); }
```

Comment:
How about renaming this to `isExpandable()` now, that seems more intuitive.

Reply:
Hmm, personally `isOpenEnded` seems clearer to me as it a pretty clear property of some range/window to me. `isExpandable` sounds like one could do something with the `View` to make it expandable, but it is really a fixed property.